### PR TITLE
Fix missing x callout custom callout

### LINKED ISSUE #26303: [Bug] Can't access xAxisCalloutData property for ICustomizedCalloutData in LineChart
### Environment Information

- **Package version(s)**: not provided

### Describe the issue:

I want to display a custom HoverCard in my LineChart, so I pass in a customHoverChart using onRenderCalloutPerDataPoint that takes props of ICustomizedCalloutData type. My ILineChartPoints have values for xAxisCalloutData and yAxisCalloutData. However, when I try to access these values like {props.values[0]?.xAxisCalloutData} and {props.values[0]?.yAxisCalloutData}, I cannot only access the xAxisCalloutData value. I can only access the yAxisCalloutData value. 

#### Actual behavior:

I can't access the xAxisCalloutData.

#### Expected behavior:

Being able to access the xAxisCalloutData value.

### If applicable, please provide a codepen repro:

[https://codepen.io/itsein/pen/qByrGNj](https://codepen.io/itsein/pen/qByrGNj)

image
In the HoverCard, "xtext2" should also appear.

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) Outlook Web

### PR DESCRIPTION
**Pull request by Shubhabrata08**
## Previous Behavior

image

## New Behavior

image

Codepen Link for Repro purpose:
https://codepen.io/itsein/pen/qByrGNj

## Related Issue(s)

- Fixes #26303

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/566e507c8882f76d2848dbec7785a317fa9c9982'>566e507c8882f76d2848dbec7785a317fa9c9982</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/utilities/utilities.ts [hunk 1]
```diff
@@ -502,6 +502,7 @@ export function calloutData(values: (ILineChartPoints & { index?: number })[]) {
       legend: string;
       y: number;
       color: string;
+      xAxisCalloutData?: string;
```
Comment (AtishayMsft) [inline packages/react-charting/src/utilities/utilities.ts:505]: Could you add an example also showing how to use this property.
The examples are in packages/react-examples/src/react-charting/.

You can add it to LineChart/LineChart.CustomAccessibility example

Comment (Shubhabrata08) [reply packages/react-charting/src/utilities/utilities.ts:505]: Sure... So an example depicting use of a custom callout containing xAxisCallout as well...

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/src/utilities/utilities.ts (hunk 1 above)
Comment (AtishayMsft) [reply packages/react-charting/src/utilities/utilities.ts:505]: The example is mentioned in the codepen link attached in the PR description

**Pull request merged**